src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -68,52 +98,190 @@ export const updateTrackStatesAndSave: ActionCreator<
   browserStorageService.saveTrackStates(trackStates);
 };
 
+export const getTracksStateInfo = (
+  genomeId: string,
+  trackCategories: GenomeTrackCategory[]
+) => {
+  const tracksInfo: TrackConfigs = {};
```

Comment:
First, variable name.

And second, it seems that what you are doing in this function is just creating a map of default track configs. Why don't you return it as `{ [trackName]: TrackConfig }`; why do you create a `{ tracks: { [trackName]: TrackConfig } }` object before returning it? Let the function that calls this one do this job.

Reply:
because I destructure the array where it is called. So its either here or there we need to add. It makes sense we add it here.